api/src/app.py

```diff
@@ -75,6 +75,11 @@ def configure_app(app: APIFlask) -> None:
     app.config["BASE_RESPONSE_SCHEMA"] = response_schema.ResponseSchema
     app.config["HTTP_ERROR_SCHEMA"] = response_schema.ErrorResponseSchema
     app.config["VALIDATION_ERROR_SCHEMA"] = response_schema.ErrorResponseSchema
+    app.config["SWAGGER_UI_CSS"] = '/static/swagger-ui.min.css'
+    app.config["SWAGGER_UI_BUNDLE_JS"] = "/static/swagger-ui-bundle.js"
+    app.config["SWAGGER_UI_STANDALONE_PRESET_JS"] = "/static/swagger-ui-standalone-preset.js"
+    # Removing because the server dropdown has accessibility issues.
+    app.config["SERVERS"] = "."
```

Comment:
Will this break the ability to use OpenAPI? The server info is currently automatically set for us, but if we manually set it, that might break querying entirely through the UI.

Reply:
I tested that it doesn't:

![image](https://github.com/HHS/simpler-grants-gov/assets/512243/46f260de-ca64-4438-ae09-460623073f6a)

Reply:
I'm going to review the API Flask code to make sure there aren't side effects for this.

Reply:
Its not going to be in APIFlask, probably something in apispec. We probably can't tell if this will be an issue until it is in dev as the server logic is going to be a lot simpler locally